api/swagger.yaml

```diff
@@ -1866,6 +1866,41 @@ definitions:
           "WorkDir": "/var/lib/docker/overlay2/ef749362d13333e65fc95c572eb525abbe0052e16e086cb64bc3b98ae9aa6d74/work"
         }
 
+  StorageDriver:
+    description: |
+      Information about the storage driver used to store the container's and
+      image's filesystem.
+    type: "object"
+    x-go-name: "Driver"
+    required: [Type, Name]
+    properties:
+      Type:
+        description: "Type of storage driver."
+        type: "string"
+        enum: ["snapshotter", "graph-driver"]
```

Comment:
I'm considering we could keep the GraphDriver .. for graphdrivers, and omit the `GraphDriver` field entirely when snapshotters are used..